### PR TITLE
Documentation describing migration to single mongoDB instance

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
Issues: https://tracker.mender.io/browse/MEN-1958

### CODE REVIEW

File: 07.Administration/06.Upgrading/docs.md [hunk 1]
```diff
@@ -42,6 +42,19 @@ These can sometimes happen due to device decommissioning.
...
 in the [Troubleshooting](../../troubleshooting/mender-server) chapter.
 
+## Migrating to single mongoDB instance.
+
+To reduce the number of mongoDB instances in the setup, first backup existing data and volumes.
+Then perform the upgrade. In the new setup, there are no mender-mongo-\* services.
+Instead, there is only one mender-mongo service which is keeping all the databases.
+After upgrading, but before starting new services, run the `migrate-db` script from migration directory.
+This script will restore data dumped from mender-mongo-\* services to the single mender-mongo instance.
+The `migrate-db` script will aslo remove old database volumes.
```
Comment (mchalski) [inline 07.Administration/06.Upgrading/docs.md]: also

Comment (kjaskiewiczz) [reply 07.Administration/06.Upgrading/docs.md]: fixed

File: 07.Administration/06.Upgrading/docs.md [hunk 2]
```diff
@@ -42,6 +42,19 @@ These can sometimes happen due to device decommissioning.
 You can find instructions on how to clean up the deviceauth database
 in the [Troubleshooting](../../troubleshooting/mender-server) chapter.
 
+## Migrating to single mongoDB instance.
+
+To reduce the number of mongoDB instances in the setup, first backup existing data and volumes.
```
Comment (drewmoseley) [inline 07.Administration/06.Upgrading/docs.md]: Should we provide details on how to do this?

File: 07.Administration/06.Upgrading/docs.md [hunk 3]
```diff
@@ -42,6 +42,19 @@ These can sometimes happen due to device decommissioning.
...
+Then perform the upgrade. In the new setup, there are no mender-mongo-\* services.
```
Comment (drewmoseley) [inline 07.Administration/06.Upgrading/docs.md]: Same question.  how do they perform the upgrade?

I would start a new paragraph with the second sentence here and word this:
"Note that with the new versions, there are no mender-mongo-* services; instead there is a single mender-mongo service which contains all the databases.

File: 07.Administration/06.Upgrading/docs.md [hunk 4]
```diff
@@ -42,6 +42,19 @@ These can sometimes happen due to device decommissioning.
...
+Instead, there is only one mender-mongo service which is keeping all the databases.
+After upgrading, but before starting new services, run the `migrate-db` script from migration directory.
```
Comment (drewmoseley) [inline 07.Administration/06.Upgrading/docs.md]: Does it make sense to show a run of the migration script with the shell output?

File: 07.Administration/06.Upgrading/docs.md (hunk 2 above)
Comment (kjaskiewiczz) [reply 07.Administration/06.Upgrading/docs.md]: How to backup? We are providing them already.

File: 07.Administration/06.Upgrading/docs.md (hunk 3 above)
Comment (kjaskiewiczz) [reply 07.Administration/06.Upgrading/docs.md]: They should perform the upgrade as described in this chapter.

File: 07.Administration/06.Upgrading/docs.md (hunk 4 above)
Comment (kjaskiewiczz) [reply 07.Administration/06.Upgrading/docs.md]: It's provided later on.

File: 07.Administration/06.Upgrading/docs.md (hunk 3 above)
Comment (kjaskiewiczz) [reply 07.Administration/06.Upgrading/docs.md]: I've updated the description.

**Comment by drewmoseley** [comment]
@estenberg @maciejmrowiec any comments before this is merged?

**Pull request merged**